store/sqlstore/channel_store.go

```diff
@@ -2465,48 +2570,84 @@ func (s SqlChannelStore) GetMembersForUserWithPagination(teamId, userId string,
 	return dbMembers.ToModel(), nil
 }
 
-func (s SqlChannelStore) AutocompleteInTeam(teamId string, term string, includeDeleted bool) (model.ChannelList, error) {
-	deleteFilter := "AND Channels.DeleteAt = 0"
-	if includeDeleted {
-		deleteFilter = ""
+func (s SqlChannelStore) GetTeamMembersForChannel(channelID string) ([]string, error) {
+	teamMemberIDs := []string{}
+	if err := s.GetReplicaX().Select(&teamMemberIDs, `SELECT tm.UserId
+		FROM Channels c, Teams t, TeamMembers tm
```

Comment:
Can we try and use squirrel in this and the rest of the queries as well? Happy to create some hacktoberfest tickets for them when this gets merged if preferred :)

Reply:
I think this is simple enough as a raw query. Unless there are various if conditions while constructing the query, I prefer raw SQL queries. :)